src/org/apache/wml/WMLAElement.java

```diff
@@ -17,10 +17,9 @@
 package org.apache.wml;
 
 /**
- * <p>The interface is modeled after DOM1 Spec for HTML from W3C.
+ * The interface is modeled after DOM1 Spec for HTML from W3C.
  * The DTD used in this DOM model is from 
- * <a href="http://www.wapforum.org/DTD/wml_1.1.xml">
- * http://www.wapforum.org/DTD/wml_1.1.xml</a></p>
+ * <a href="http://www.wapforum.org/DTD/wml_1.1.xml">http://www.wapforum.org/DTD/wml_1.1.xml</a>
```

Comment:
https

Reply:
I've replaced all `http://www.wapforum.org/DTD/wml_1.1.xml` with `https://www.wapforum.org/DTD/wml_1.1.xml` is the next commit. Worth noting that the resulting document won't render in a modern browser so it's likely that people will need to use curl to view the content. I was surprised it loaded at all as wap and wml died a long time ago.